satellite/gracefulexit/endpoint.go

```diff
@@ -915,10 +920,29 @@ func (endpoint *Endpoint) updatePointer(ctx context.Context, originalPointer *pb
 			NodeId:   receivingNodeID,
 		}}
 	}
-	_, err = endpoint.metainfo.UpdatePiecesCheckDuplicates(ctx, string(path), originalPointer, toAdd, toRemove, true)
+	_, err = endpoint.metainfo.UpdatePiecesCheckDuplicates(ctx, path, originalPointer, toAdd, toRemove, true)
 	if err != nil {
 		return Error.Wrap(err)
 	}
 
 	return nil
 }
+
+func (endpoint *Endpoint) getValidPointer(ctx context.Context, path string, pieceNum int32, originalRootPieceID storj.PieceID) (*pb.Pointer, error) {
+	pointer, err := endpoint.metainfo.Get(ctx, path)
+	// TODO we don't know the type of error
+	if err != nil {
+		return nil, Error.New("pointer path %v no longer exists.", path)
```

Comment:
These errors aren't really about the errors, so mentioning "pointer path" here would be confusing. Use "pointer %q no longer exists" or similar. Also keep the original error to see what the actual error was.